ddtrace/contrib/django/db.py

```diff
@@ -41,95 +43,46 @@ def unpatch_db():
     connections.all = all_connections
     delattr(connections, ALL_CONNS_ATTR)
 
+
```

Comment:
🙇

Reply:
@brettlangdon Can you explain if this is still an issue?

Reply:
no, this isn't an issue, praising Luca for adding the missing extra newline :)